cmd/snap-confine/snap-confine.c

```diff
@@ -63,6 +64,26 @@ void sc_maybe_fixup_permissions()
 	}
 }
 
+// sc_maybe_fixup_udev will remove incorrectly created udev tags
+// that cause libudev on 16.04 to fail with "udev_enumerate_scan failed".
+// See also:
+// https://forum.snapcraft.io/t/weird-udev-enumerate-error/2360/17
+void sc_maybe_fixup_udev()
+{
+	glob_t glob_res SC_CLEANUP(globfree) = {
+	.gl_pathv = NULL};
```

Comment:
Can you also set .gl_pathc and .gl_offs to 0?

Reply:
Hmm, aren't those set by the C standard? We set one field, the rest is initialized to zero.